src/coreclr/nativeaot/docs/containers.md

```diff
@@ -1,90 +0,0 @@
-# Building native AOT apps in containers
-
-Containers are a useful tool to build any kind of code, including native AOT. There are a few patterns for doing that.
-
-## Cloud native apps
-
-Native AOT apps can be built in containers (with multi-stage build) just like regular (CoreCLR) apps. SDK `aot` images provide the additional native toolchain dependencies required to build native AOT apps.
-
-Search for `aot` images at [dotnet/nightly/sdk](https://mcr.microsoft.com/en-us/product/dotnet/nightly/sdk/tags).
```

Comment:
Similar here. I think this is the only place we have description of the nightly aot images.